exercises/phone-number/src/test/java/PhoneNumberTest.java

```diff
@@ -92,17 +103,41 @@ public void invalidWithLetters() {
 
     @Ignore("Remove to run test")
     @Test
-    public void invalidWithPunctuation() {
+    public void invalidWithPunctuations() {
         expectedException.expect(IllegalArgumentException.class);
         expectedException.expectMessage(illegalCharacterExceptionMessage);
         new PhoneNumber("123-@:!-7890");
     }
+    
+    @Ignore("Remove to run test")
+    @Test
+    public void invalidIfAreaCodeStartsWith0() {
+        expectedException.expect(IllegalArgumentException.class);
+        expectedException.expectMessage(illegalCharacterExceptionMessage);
```

Comment:
I don't think this error message very accurately describes what has gone wrong. Please change it to something which describes the problem 🙂 The same applies to the next two tests 🙂